### DISCUSSION TITLE
Arcane mage mana gem

### DISCUSSION THREAD

**Discussion by ArcticLatent** [Hekili/hekili#944] [Q&A] [answered]
If there is no mana gem in backpack hekili shows conjure mana gem in rotation but it doesnt have a feature like showing when to use the mana gem when your mana is low and evo is on cd. Am i missing something? If not can this feature be added please? Thanks

**Comment by Hekili** [comment, marked as answer]
Yes, it does.

**Reply by ArcticLatent** [reply]
It does what? It shows when to use mana gem when low on mana? I dont see it. What can be the problem.

**Reply by Hekili** [reply]
The Arcane Mage module and priority includes usage of a mana gem.  If you are not seeing a recommendation to use a mana gem when you think you should, submit a proper ticket with all the required information (including a snapshot).
